sky/packages/flx/lib/signing.dart

```diff
@@ -3,7 +3,7 @@
 // found in the LICENSE file.
 
 import 'dart:async';
-import 'dart:convert';
+import 'dart:convert' hide BASE64;
```

Comment:
Rather than hiding specific symbols, should we import this library into a namespace?